src/components/GenreFilter/index.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import "./genre-filter.css";
+
+function GenreFilter(props) {
+  const [bookGenres, setBookGenres] = useState([]);
+
+  useEffect(() => {
+    fetch("https://book-swap-api.dev.io-academy.uk/api/genres")
+      .then((res) => res.json())
+      .then((data) => {
+        setBookGenres(data.data);
+      });
+  }, []);
+
+  function setGenre(event) {
+    props.setGenreID(event.target.value);
+  }
+
+  function setGenre(event) {
+    if (props.setGenreID) {
+      props.setGenreID(event.target.value);
+    } else if (props.ClaimedGenreID) {
+      props.setClaimedGenreID(event.target.value);
+    }
```

Comment:
hmmm I don't like this. Ultimately setting a genre is really the same if it's claimed or not. The necessity for this if statement tells me that your approach to splitting out claimed/unclaimed might not be working out so well now.